http-builder-ng-core/src/main/java/groovyx/net/http/NativeHandlers.java

```diff
@@ -348,28 +354,6 @@ public static void json(final ChainedHttpConfig config, final ToServer ts) {
             }
         }
 
-        public static void transfer(final InputStream istream, final OutputStream ostream, final boolean close) {
```

Comment:
Can we resurrect this method, forward calls on to IoUtils.transfer(), and then deprecate it? I doubt anyone was using it, but it is public so it's possible someone was using it. We can let it live for another minor version or two and then kill it. We can definitely kill it for a 1.0 release.